### PR TITLE
Replace home-grown client with go-zendesk

### PR DESCRIPTION
**Pull request by antoineco**
We have 0 test while this lib does have a good coverage and seems well supported. Besides, it has all the operations we need.

### CODE REVIEW

**Comment by JeffNaef-Triggermesh** [comment]
when you say "go-zendesk" are you reffering to this -> https://github.com/MEDIGO/go-zendesk

File: go.mod [hunk 1]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/cloudevents/sdk-go/v2 v2.0.1-0.20200630063327-b91da81265fe
 	github.com/google/go-cmp v0.4.0
 	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/nukosuke/go-zendesk v0.7.6
```
Comment (antoineco) [inline go.mod:17]: @JeffNaef-Triggermesh to answer your question

**Comment by JeffNaef-Triggermesh** [comment]
or -> https://github.com/nukosuke/go-zendesk

**Comment by antoineco** [comment]
Tested and working like a charm, moving to _Ready for review_.

**Pull request merged**